docs/installation/language.md

```diff
@@ -28,9 +28,11 @@ In [1]: import helics
 
 In [2]: helics.helicsGetVersion()
 Out[2]: 'x.x.x (XXXX-XX-XX)'
-
 ```
 
+Note: If you already have a HELICS installation with the C shared library, it is possible to build the Python3 interface
+using the `interfaces/python` folder as a standalone CMake project. This can be much faster than rebuilding HELICS.
+
```

Comment:
Can you add the install command here?

Reply:
Would something like “e.g. `cmake <path to interfaces/python folder>`”  be enough to make it clear what it means by using the folder as a standalone cmake project? For a system directory install of HELICS that will work, otherwise the default CMake error messages will tell them what variables can get set to make it find a custom HELICS install.